src/Nancy.Swagger.Annotations/SwaggerObjects/AnnotatedModel.cs

```diff
@@ -16,10 +16,26 @@ public AnnotatedModel(Type type, ModelAttribute modelAttr) : base(type)
                                         .Where(pi => pi.CanWrite && pi.GetSetMethod(true).IsPublic)
                                         .Where(pi => pi.CanRead && pi.GetGetMethod(true).IsPublic);
 
-            Properties = typeProperties.Select(CreateSwaggerModelPropertyData).ToList();
+            Properties = GetSwaggerModelProperties(typeProperties);
             Description = modelAttr.Description;
         }
 
+        private IList<SwaggerModelPropertyData> GetSwaggerModelProperties(IEnumerable<PropertyInfo> properties)
+        {
+            var result = new List<SwaggerModelPropertyData>();
+            //filter out ignored properties.
+            foreach(var prop in properties)
```

Comment:
Can you do a linq expression?
Maybe something like 
```
properties
    .Where(prop => !prop.GetCustomAttribute<ModelPropertyAttribute>()?.Ignore ?? true)
    .Select(CreateSwaggerModelPropertyData).ToList();
```

Then you don't need to add a whole new function.
(Please double check my logic though)

Reply:
Sure, I just thought the method would be clearer to read, will push change shortly.